### DISCUSSION TITLE
Changing Aircraft Parameters with Thrustmaster Throttle and Joystick

### DISCUSSION THREAD

**Discussion by Vilas-jacob** [TUDelft-CNS-ATM/bluesky#472] [General]
Hello fellow developers,

I am currently working on this air traffic simulator project and would like to explore the possibility of using a Thrustmaster throttle and joystick to modify aircraft parameters such as height, speed, and headings. This inquiry is purely for learning purposes.

If anyone has experience or knowledge in integrating Thrustmaster peripherals into a simulator project, I would greatly appreciate your insights. Specifically, I'm interested in understanding the procedure or steps involved in implementing this functionality within the simulator.

Please feel free to share any resources, code snippets, or suggestions that could help me achieve this interaction between the simulator and the Thrustmaster throttle and joystick.

Thank you in advance for your contributions and expertise. I look forward to engaging in an enlightening discussion with the community.

Best regards,
Vilas

**Comment by ProfHoekstra** [comment]
Dear Vilas,

The pygame library has a good interface for joysticks. 

Best regards,
    Jacco Hoekstra
